content/docs/examples/bookinfo/index.md

```diff
@@ -93,8 +93,19 @@ To start the application, follow the instructions below corresponding to your Is
 
     *   If you are using a cluster with
```

Comment:
@rokii thanks for the contribution!  I'd prefer to just remove the automatic sidecar injection documentation almost entirely (except for the sidecar document itself) as the TOC has decided that sidecar injection will be a mandatory requirement.  Can you rework this document with that in mind?

Cheers
-steve

Reply:
@frankbu see https://groups.google.com/forum/#!topic/istio-dev/ZdMbghc7sDE.  Perhaps I read too much into that, and your position is fair.  I just believe we shouldn't document manual injection in every single example page and should lean on automatic injection for most cases.

I do believe we should continue to document manual injection in the main automatic sidecar injection page, as it is very helpful.

Cheers
-steve